examples/algorithms/cuda/include/cuda/track_finding/seeding_algorithm.hpp

```diff
@@ -16,13 +16,13 @@
 namespace traccc {
 namespace cuda {
 
-class seeding_algorithm {
-    public:
-    using input_type = host_spacepoint_container&;
-    using output_type =
-        std::pair<host_internal_spacepoint_container, host_seed_container>;
+class seeding_algorithm
+    : public algorithm<
+          std::pair<host_internal_spacepoint_container, host_seed_container>(
+              host_spacepoint_container&&)> {
```

Comment:
Are these changes related to the track parameter seeding in some way or have they snuck into this PR?

Reply:
They snuck.